thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/route.js

```diff
@@ -354,6 +355,55 @@ export default Route.extend({
     return fetch(`/detection-job/update/filter/${id}`, postProps('')).then(checkStatus);
   },
 
+  /**
+   * This concurrency task fetches anomaly performance metrics and data for all anomalies which
+   * would not be included in the notification set under the user-selected tuning settings.
+   * @method triggerTuningSequence
+   * @param {Object} configObj - the user-selected type and value of tuning severity thresholds
+   * @return {undefined}
+   */
+  triggerTuningSequence: task(function * (configObj) {
+    try {
```

Comment:
do we need the try around the whole function here? ideally, this can only error out at the fetching step

Reply:
True - no need to wrap the entire function. I'll move it down to where the fetching begins.